publisher/internal/analytics/ethereum/operations.go

```diff
@@ -457,7 +456,7 @@ func (p Position) areTokensSet() bool {
 	return (!strings.EqualFold(p.Token0.Address, "") && !strings.EqualFold(p.Token1.Address, ""))
 }
 
-func (p Position) isOrderCorrect() bool {
+func (p Position) isStableOrNativeInCorrectPosition() bool {
```

Comment:
This naming does not resolve domain-specific knowledge, and is overly verbose, thus it is worse than what was before.

I think a better name would be: `IsToken0Stable`. 

This will open up opportunities to check for other stable-coins in the future. 

Although it still requires some amount of domain-specific knowledge(at least our definition of "correct order" which seems arbitrary), however, at least it is more in-line with the purpose of the function.